test/rules/__init__.py

```diff
@@ -50,7 +50,7 @@ class BaseRuleTestCase(BaseTestCase):
 
     def setUp(self):
         """Setup"""
-        self.collection = RulesCollection()
+        self.collection = RulesCollection([])
```

Comment:
It's defaulted, why not keep it `RulesCollection()`? Keep the "no excluded rule" logic clean (in the constructor).

Also for all other places with this change of course